### PR TITLE
refactor: Switch legacykms to localkms when it's ready

### LINKED ISSUE #309: Switch legacykms to localkms when it's ready

### PR DESCRIPTION
**Pull request by fqutishat**
closes #309

Signed-off-by: Firas Qutishat <firas.qutishat@securekey.com>

### CODE REVIEW

File: cmd/vc-rest/startcmd/start.go [hunk 1]
```diff
@@ -395,13 +402,17 @@ func startEdgeService(parameters *vcRestParameters, srv server) error {
...
+
 	vcService, err := vc.New(&operation.Config{StoreProvider: edgeServiceProvs.provider,
 		KMSSecretsProvider: edgeServiceProvs.kmsSecretsProvider,
-		EDVClient:          edv.New(parameters.edvURL, edv.WithTLSConfig(tlsConfig)),
-		LegacyKMS:          legacyKMS,
-		KeyManager:         localKMS,
-		VDRI:               vdri, HostURL: externalHostURL, Mode: parameters.mode, Domain: parameters.blocDomain,
-		TLSConfig: tlsConfig})
+		EDVClient:          edv.New(parameters.edvURL, edv.WithTLSConfig(&tls.Config{RootCAs: rootCAs})),
+		LegacyKMS:          legacyKMS, KeyManager: localKMS,
```
Comment (troyronda) [inline cmd/vc-rest/startcmd/start.go:413]: why still legacyKMS

Comment (fqutishat) [reply cmd/vc-rest/startcmd/start.go:413]: @DRK3 @Baha-sk add your comments for JWE

Comment (Baha-sk) [reply cmd/vc-rest/startcmd/start.go:413]: JWE still uses LegacyKMS @troyronda

File: pkg/restapi/vc/operation/operations.go [hunk 2]
```diff
@@ -145,6 +138,11 @@ type EDVClient interface {
 	QueryVault(vaultID string, query *models.Query) ([]string, error)
 }
 
+type keyManager interface {
```
Comment (Baha-sk) [inline pkg/restapi/vc/operation/operations.go:141]: Probably rename to `ExportableKeyManager`

**Pull request merged**